### ISSUE TITLE
FR: Disable folder expander if no subfolders under the folder

### ISSUE THREAD

**Issue by gerroon** [dani-garcia/bitwarden_rs#523]
Hi

It would be nice  the folder  does not display a "triangle" folder expander icon if it has no "subfolder" At the moment it is a bit hard to tell if a folder entry has subfolders or not since all folders have either sideways looking triangle or downwards triangle regardless of they have children folders under them or not. 

This is only obvious if the folder is expanded and one can see the subfolders.

Basically if a folder has no subfolder it should not have an expander icon.

thanks

**Comment by dani-garcia** [comment]
This is mostly a client issue, so we can't do anything ourselves to solve it.

There is a feature request already in upstreams forum, you might want to subscribe there: https://community.bitwarden.com/t/no-caret-on-no-children-folders/5736/2

Closing this as there's nothing for us to do.
